artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ActiveMQServerImpl.java

```diff
@@ -814,7 +815,7 @@ public void run() {
    }
 
    private static void checkCriticalAnalyzerLogging() {
-      Logger criticalLogger = LoggerFactory.getLogger("org.apache.activemq.artemis.utils.critical");
+      Logger criticalLogger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
This changes the logger name used for the check to the wrong value, the one for ActiveMQServerImpl (and so also duplicates the class-level logger), rather than "org.apache.activemq.artemis.utils.critical" used before, which is what is also referenced in the log4j2.properties config.

Should be changed to a value derived from the declaration of the original logger elsewhere (maybe declared as a constant).